class.eventcalendar.plugin.php

```diff
@@ -304,7 +319,7 @@ private function formatEventCalendarDate($eventDate = '0000-00-00', $includeIcon
 
         return sprintf(
             t('EventCalendar.DateMarkup', '<div class="EventCalendarDate">%2$s On %1$s</div>'),
-            strftime(t('EventCalendar.DateFormat', '%A, %e. %B %Y'), strtotime($eventDate)),
+            strftime(t('EventCalendar.DateFormat', '%A, %e %B %Y'), strtotime($eventDate)),
```

Comment:
The same as above: that is translatable so that source code doesn't need to be changed